manifest.md

```diff
@@ -0,0 +1,261 @@
+
+  The manifests field contains a list of manifests for specific platforms.
+
+  Fields of a object in the manifests list are:
+
+  - **`mediaType`** *string*
+
+    The MIME type of the referenced object. This will generally be
+    `application/vnd.oci.image.manifest.v1+json`, but it could also
+    be `application/vnd.oci.image.manifest.v1+json` if the manifest
+    list references a legacy schema-1 manifest.
+
+  - **`size`** *int*
+
+    The size in bytes of the object.
+    This field exists so that a client will have an expected size for the content before validating.
+    If the length of the retrieved content does not match the specified length, the content should not be trusted.
+
+  - **`digest`** *string*
+
+    The digest of the content, as defined by the [Registry V2 HTTP API Specificiation](https://docs.docker.com/registry/spec/api/#digest-parameter).
+
+  - **`platform`** *object*
+
+    The platform object describes the platform which the image in the manifest runs on.
+    A full list of valid operating system and architecture values are listed in the [Go language documentation for `$GOOS` and `$GOARCH`](https://golang.org/doc/install/source#environment)
+
+    - **`architecture`** *string*
+
+        The architecture field specifies the CPU architecture, for example `amd64` or `ppc64le`.
+
+    - **`os`** *string*
+
+        The os field specifies the operating system, for example `linux` or `windows`.
+
+    - **`os.version`** *string*
+
+        The optional os.version field specifies the operating system version, for example `10.0.10586`.
+
+    - **`os.features`** *array*
+
+        The optional os.features field specifies an array of strings, each listing a required OS feature (for example on Windows `win32k`).
+
+    - **`variant`** *string*
+
+        The optional variant field specifies a variant of the CPU, for example `armv6l` to specify a particular CPU variant of the ARM CPU.
+
+    - **`features`** *array*
+
+        The optional features field specifies an array of strings, each listing a required CPU feature (for example `sse4` or `aes`).
+
+## Example Manifest List
+
+*Example showing a simple manifest list pointing to image manifests for two platforms:*
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.list.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 7143,
+      "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f",
+      "platform": {
+        "architecture": "ppc64le",
+        "os": "linux",
+      }
+    },
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 7682,
+      "digest": "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270",
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux",
+        "features": [
+          "sse4"
+        ]
+      }
+    }
+  ]
+}
```

Comment:
> should there be some kind of URI pointing to where the referenced image can be downloaded from? (optional?)

This defeats the purpose of providing content addresses. Pushing down the fetch content also over-couples the description of the image from the distribution of the image.

> should we allow for referenced images to be "in-lined"? It would be really nice if we could "export" a fat-manifest and share it by just passing around a single file instead of a set of them. People will probably end up tar'ing them up anyway, so we might as well formalize it.

This was considered. However, it burdens verification of the "in-lined" manifest. The identity becomes dependent on the ability to re-serialize the embedded format. Re-serialization should be avoided at all costs in a robust content-addressable system.